internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -93,7 +96,7 @@ func criSysconfig(t *Target, data interface{}) error {
 		if err != nil {
 			// If the deferred function has any return values, they are discarded when the function completes
 			// https://golang.org/ref/spec#Defer_statements
-			fmt.Println("Could not delete the cri.d config path")
+			fmt.Println("Could not delete the temporary cri.d config path")
```

Comment:
the criSysconfig function is rightfully called, but it doesn't do anything interesting for us in the current code.
This function should be refactored to instead copy sysconfig from the package as a default sysconfig for crio in `/etc/sysconfig` . This way, the node migration will automatically ensure that sysconfig is always the default one, and the local configuration is rightfully applied with criConfigure.